npm-policy.md

```diff
@@ -0,0 +1,73 @@
+# The npm policy
+
+We use [npmjs](https://www.npmjs.com/) to publish our JavaScript packages. This document outlines how we use it.
+
+- [Team permissions](#team-permissions)
+- [Minimal viable `package.json`](#minimal-viable-packagejson)
+- [Naming conventions](#naming-conventions)
+- [2FA](#2fa)
+
+### Team permissions
+
+We do not use an @OrbitDB team for scoping packages, due to issues with republishing scoped packages and the long nature of `@orbitdb/orbit-db` as an annoying name to type. Instead, publish under your own name.
+
+However; ensure that you grant publishing rights to another core contributor. The way to do this is to use [`npm owner add @user`](https://docs.npmjs.com/cli/owner).
```

Comment:
Since npm lets package owners do anything, should there be a policy or process of authorizing the user before granting them ownership?

Reply:
I was letting this one be a soft point. They must already be a maintainer of the repository. Beyond that, leave it up to the community's discretion.